.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="11" />
+    <bytecodeTargetLevel target="15" />
```

Comment:
accidental change? while commiting only commit the files that you have changed. Android studio usually changes .idea files automatically.

Reply:
I am sorry I don't understand why is this happening maybe it's because I committed after pulling upstream